client/src/pages/home/helpers.ts

```diff
@@ -0,0 +1,95 @@
+import { IIssue, IPitch, IUser, IUserAggregate } from 'ssw-common';
+
+import { pitchStatusEnum } from '../../utils/enums';
+
+const sswEstablishedYear = 1995;
+
+const getYearsSinceSSWEstablished = (): string[] => {
```

Comment:
This is nitpicky but later in your sorting, you cast the .getFullYear() to a string to compare it with one of the values you create here. Why not make this a return type of `number[]` that way you don't have to do that cast later.